packages/openapi-2-kong/src/declarative-config/plugins.js

```diff
@@ -25,12 +25,16 @@ export function generatePlugins(item: Object, generator: GeneratorFn): Array<DCP
 export function generatePlugin(key: string, value: Object): DCPlugin {
   const plugin: DCPlugin = {
     name: value.name || getPluginNameFromKey(key),
+    config: value.config ? value.config : {},
```

Comment:
if `value.config` is not defined, fallback to `{}`

```suggestion
    config: value.config || {},
```